supervisor/pip.sls

```diff
@@ -2,7 +2,7 @@ supervisor:
   pip.installed:
     - name: supervisor==3.0
     - require:
-      - pkg: python-pip
+      - pip: pip
```

Comment:
Looks like everywhere we require `supervisor.pip`, we also include `project.venv`, which itself includes `python`, so it's probably fine. The docs on the `include` statement are pretty sparse: https://docs.saltstack.com/en/latest/ref/states/highstate.html#include-declaration

In any case, I say :ship: